zenodo/config.py

```diff
@@ -840,6 +842,10 @@ def _(x):
 
 # Search
 # ======
+#: Default ElasticSearch host
+SEARCH_ELASTIC_HOSTS = [
+    {"host": ZENODO_DOCKER_HOST}
+]
```

Comment:
Not needed since it's localhost by default:

https://github.com/inveniosoftware/invenio-search/blob/master/invenio_search/config.py#L38 (None from default Invenio config)
https://github.com/inveniosoftware/invenio-search/blob/master/invenio_search/ext.py#L107 (instantiated with None)
http://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch